data/core.yaml

```diff
@@ -552,284 +552,158 @@ en:
-      or clicking on the first node drawn to close the shape.
-
-      ### Deleting
-
-      If a building is entirely incorrect - you can see that it doesn't exist in satellite
-      imagery and ideally have confirmed locally that it's not present - you can delete
-      it, which removes it from the map. Be cautious when deleting features -
-      like any other edit, the results are seen by everyone and satellite imagery
-      is often out of date, so the building could simply be newly built.
-
-      You can delete a building by clicking on it to select it, then pressing the 'Delete'
-      key, or right-clicking it and then clicking the trash can icon.
-    relations: |
-      # Relations
-
-      A relation is a special type of feature in OpenStreetMap that groups together
-      other features. For example, two common types of relations are *route relations*,
-      which group together sections of road that belong to a specific freeway or
-      highway, and *multipolygons*, which group together several lines that define
-      a complex area (one with several pieces or holes in it like a donut).
-
-      The group of features in a relation are called *members*. In the bottom of the
-      sidebar, you can see which relations a feature is a member of, and click on a
-      relation there will select it. When the relation is selected, you can see all of
-      its members listed in the sidebar and highlighted on the map.
-
-      For the most part, iD will take care of maintaining relations automatically
-      while you edit. The main thing you should be aware of is that if you delete a
-      section of road to redraw it more accurately, you should make sure that the
-      new section is a member of the same relations as the original.
-
-      ## Editing Relations
-
-      If you want to edit relations, here are the basics.
-
-      To add a feature to a relation, select the feature, click the "+" button in the
-      "All relations" section of the sidebar, and select or type the name of the relation.
-
-      To create a new relation, select the first feature that should be a member,
-      click the "+" button in the "All relations" section, and select "New relation...".
-
-      To remove a feature from a relation, select the feature and click the trash
-      button next to the relation you want to remove it from.
-
-      You can create multipolygons with holes using the "Merge" tool. Draw two areas (inner
-      and outer), hold the Shift key and click on each of them to select them both, and then
-      press the 'C' shortcut key. Other option is to select both, and then right-click one
-      of the and click the "Merge" (+) button.
+    help:
+      title: Help
+      welcome: "Welcome to the iD editor for [OpenStreetMap](https://www.openstreetmap.org/). With this editor you can update OpenStreetMap right from your web browser."
+      open_data_h: "### Open Data"
```

Comment:
Perhaps as tail work, can we define the heading levels somewhere outside these strings? That would make it less error-prone to translate the help UI.

Reply:
Good suggestion, I removed the markdown from the titles, but left it here, and it feels messy. I'll think of a way.